src/material-examples/button-overview/button-overview-example.ts

```diff
@@ -1,6 +1,10 @@
 import {Component} from '@angular/core';
 
-
+/**
+ * @title Basic buttons
+ * @id button-overview
+ * @component ButtonOverviewExample
```

Comment:
I first tried `require(file)` and looking at the exported names but caused runtime issues. I spoke w/ @DevVersion and we have a plan to do this in typescript parser.